website/docs/intro.mdx

```diff
@@ -31,8 +31,6 @@ import Json from "@site/static/toc.json";
 
 test 1234
 
-hello
-
 ```proto-spc
```

Comment:
Just curious, `proto-spec`? What is `spc`?

Reply:
Oh, yes. I was thinking of "special" or something like that since it deviates from the usual `protobuf` code language.

Would it be better if we downright use `protosaurus` instead?

Reply:
OK, I have changed it to `protosaurus` and a TODO in the plugin to specify a better name. We can do it at any point of time in the future.